web_src/js/features/user-auth-webauthn.js

```diff
@@ -169,12 +155,16 @@ function detectWebAuthnSupport() {
 }
 
 export function initUserAuthWebAuthnRegister() {
-  if ($('#register-webauthn').length === 0) {
+  document.getElementById('webauthn-error')?.classList.add('hide');
+
+  const elRegister = document.getElementById('register-webauthn');
+  if (!elRegister) {
     return;
   }
 
-  $('#webauthn-error').modal({allowMultiple: false});
-  $('#register-webauthn').on('click', (e) => {
+  document.getElementById('webauthn-error')?.classList.add('hide');
```

Comment:
I think the old logic is better:

1. check if `#register-webauthn` exists, if not, return.
2. Just do `document.getElementById('webauthn-error').xxx`, no need to use `?`

Reply:
Safer to use the `?` so I'll leave it - but I've removed the duplicate.

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/22697/commits/93ba2326687450727181bfae3024e926a0934efb.